runatlantis.io/docs/server-configuration.md

```diff
@@ -456,6 +456,21 @@ Values are chosen in this order:
   ```
   Log level. Defaults to `info`.
 
+### `--markdown-template-overrides-dir`
+  ```bash
+  atlantis server --markdown-template-overrides-dir="path/to/templates/"
+  ```
+  Directory where Atlantis will read in overrides for markdown templates used to render comments on pull requests.
+  Markdown template overrides may be specified either in individual files, or all together in a single file. All template
+  override files _must_ have the `.tmpl` extension, otherwise they will not be parsed.
+
+  Markdown templates which may have overrides can be found [here](../../server/events/templates)
+
+  Please be mindful that settings like `--enable-diff-markdown-format` depend on logic defined in the templates. It is
+  possible to diverge from expected behavior, if care is not taken when overriding default templates.
+
+  Defaults to `~/.markdown_templates/`
```

Comment:
What is `~` here ? is it the atlantis home directory ? If so, let's be explicit here.

Reply:
Would explaining what `~/` is be sufficient (i.e. `'~/' represents the atlantis home dir`) or something like `/<atlantis_home_dir>/.markdown_templates/` ?

`--data-dir` uses the same notation in its description (just refers to `~/.atlantis`, but without explanation of what `~/` is.

Should I update `--data-dir` to reflect any explanation or expansion of `~/` I do here?

Reply:
This should be fine.

```suggestion
  Defaults to the atlantis home directory `/home/atlantis` in `$HOME/.markdown_templates/`
```